packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_home/views/home_page.dart

```diff
@@ -123,3 +118,33 @@ class NavMenuItem {
   final RouteDataModel route;
   final String routePath;
 }
+
+class BranchContainer extends StatelessWidget {
+  const BranchContainer({
+    super.key,
+    required this.currentIndex,
+    required this.children,
+    });
+
+    final int currentIndex;
+    final List<Widget> children;
+
+    @override
+    Widget build(BuildContext context) {
+      return Stack(
+        children: children.mapIndexed(
+          (int index, Widget navigator) {
+            return Opacity(
+              opacity: index == currentIndex ? 1 : 0,
```

Comment:
Create a local variable for `index == currentIndex` and use it everywhere in this mapper body.